cpp/ast.py

```diff
@@ -1428,6 +1428,7 @@ def _get_bases(self):
     def _get_class(self, class_type, templated_types):
         class_name = None
         class_token = self._get_next_token()
+        name_tokens = []
```

Comment:
It is possible that the first branch of the if statement is taken. For the case I am running where it fails, ```class_token.token_type) == SYNTAX```